R/pkg/R/SQLContext.R

```diff
@@ -214,18 +214,45 @@ setMethod("toDF", signature(x = "RDD"),
 #' sc <- sparkR.init()
 #' sqlContext <- sparkRSQL.init(sc)
 #' path <- "path/to/file.json"
-#' df <- jsonFile(sqlContext, path)
+#' df <- read.json(sqlContext, path)
 #' }
 
-jsonFile <- function(sqlContext, path) {
+read.json <- function(sqlContext, ...) {
```

Comment:
Keep the original signature.
do not use var arg for paths. Use a vector of string for paths, This is more R-like style.